### PR TITLE
Add ansible tmp dir to fix running ansible-doc in importer

### PR DESCRIPTION
**Pull request by awcrosby**
Error occurs in openshift env, a permission error on creating the tmp dir

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -55,10 +55,12 @@ RUN set -ex; \
 RUN set -ex; \
     mkdir -p /var/lib/pulp/artifact \
              /var/lib/pulp/tmp \
+             /.ansible/tmp \
     && chown -R ${USER_NAME}:${USER_GROUP} \
         /app \
         /venv \
         /var/lib/pulp \
+        /.ansible/tmp \
```
Comment (cutwater) [inline Dockerfile:63]: Can you configure it to use different directory?
I think it should be declared as a VOLUME.

Comment (awcrosby) [reply Dockerfile:63]: in galaxy-dev we added `ansible.cfg` to make it use `/tmp/ansible`, we can do the same and make that a VOLUME

Comment (cutwater) [reply Dockerfile:63]: Please do it.

Comment (awcrosby) [reply Dockerfile]: I made the updates - I only added the volume to the pulp worker - let me know how these changes look

File: dev/docker-compose.yml [hunk 2]
```diff
@@ -48,6 +48,7 @@ services:
       - "./common/settings.py:/etc/pulp/settings.py:z"
       - "${COMPOSE_CONTEXT}/..:/src:z"
       - "pulp_artifact:/var/lib/pulp/artifact"
+      - "ansible_tmp:/tmp/ansible"
```
Comment (cutwater) [inline dev/docker-compose.yml:51]: This is unnecessary. Normally you don't need persistent volume for temporary files.

Comment (awcrosby) [reply dev/docker-compose.yml]: updated

**Pull request merged**